flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/gateway/local/LocalExecutorITCase.java

```diff
@@ -229,7 +229,6 @@ public void testCompleteStatement() throws Exception {
 		final SessionContext session = new SessionContext("test-session", new Environment());
 
 		final List<String> expectedTableHints = Arrays.asList(
-			"TABLE",
```

Comment:
This is removed due to the new `SqlAdvisor` now does not default ignore case when matches hints. alternative option is to pass in the ParserConfig to the SqlAdvisor - But the ParserConfig also has ignoreCase set to false, which is confusing to me why this test allows "TABLE" in the first place.

Reply:
you mean make the parserConfig to ignoreCase? I am not sure that would work since it's also used elsewhere. If you are fine with only case sensitive advisor results, can we keep it this way?